controllers/appgroup_controller_suite_test.go

```diff
@@ -34,49 +31,68 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
-var cfg *rest.Config
-var k8sManager ctrl.Manager
-var k8sClient client.Client
-var testEnv *envtest.Environment
-var tempChartStoreTargetDir string
-var portForwardStagingRepoURL string = "http://127.0.0.1:8080"
-var inClusterstagingRepoURL string = "http://orkestra-chartmuseum.orkestra:8080"
+const (
+	portForwardStagingRepoURL = "http://127.0.0.1:8080"
+	inClusterstagingRepoURL   = "http://orkestra-chartmuseum.orkestra:8080"
+)
+
+var (
+	k8sClient               client.Client
+	tempChartStoreTargetDir string
+	testEnv                 *envtest.Environment
+
+	AfterEach  = ginkgo.AfterEach
+	BeforeEach = ginkgo.BeforeEach
+	By         = ginkgo.By
+	Context    = ginkgo.Context
+	Describe   = ginkgo.Describe
+	It         = ginkgo.It
+
+	BeNil        = gomega.BeNil
+	BeTrue       = gomega.BeTrue
+	Equal        = gomega.Equal
+	Eventually   = gomega.Eventually
+	Expect       = gomega.Expect
+	HaveOccurred = gomega.HaveOccurred
```

Comment:
This might cause confusion for newer devs coming from a ginkgo background. Let's keep ginkgo and gomega as a `.` import for consistency, unless you have a good reason to do what you are doing here?